docker/local/docker-compose.yml

```diff
@@ -0,0 +1,77 @@
+version: '3.5'
+
+services:
+  ## INITIALIZE GENESIS AND SECRETS
+  init:
```

Comment:
The genesis file outputted seems to be missing or mis-interpolating the appropriate validator keys.

Reply:
How did you verify it?